.travis.yml

```diff
@@ -8,8 +8,10 @@ env:
   - RUBY_GC_HEAP_GROWTH_MAX_SLOTS=300000
   - RUBY_GC_HEAP_INIT_SLOTS=600000
   - RUBY_GC_HEAP_GROWTH_FACTOR=1.25
+  - RAILS_ENV=test
+  - PARALLEL_TEST_PROCESSORS=2
   matrix:
-  - TEST_SUITE=vmdb
+  - TEST_SUITE=vmdb PARALLEL='^spec/(controllers|helpers|initializers|javascripts|lib|mailers|models|presenters|requests|routing|services|task_helpers|tools|views)'
```

Comment:
Putting a prior comment back here since a force push collapsed it:

In other words, I need to do this because I want these directories but want to exclude spec/replication, spec/migrations, and spec/automation.

I don't see why we can't move these directories by suite:

spec/migrations
spec/replication
spec/automation
spec/vmdb/controllers
spec/vmdb/helpers
spec/vmdb/initializers
spec/vmdb/javascripts
spec/vmdb/lib
spec/vmdb/mailers
spec/vmdb/models
spec/vmdb/presenters
spec/vmdb/requests
spec/vmdb/routing
spec/vmdb/services
spec/vmdb/task_helpers
spec/vmdb/tools
spec/vmdb/views
This line would then become: TEST_SUITE=vmdb PARALLEL='^spec/vmdb'

We'd still have the problem of rake test, rake, rspec all still running ALL tests but at least we can more easily run tests.

Thoughts?